tools/docker-compose/bootstrap_development.sh

```diff
@@ -17,8 +17,8 @@ ansible -i "127.0.0.1," -c local -v -m wait_for -a "host=${RABBITMQ_HOST} port=5
 #/etc/init.d/ssh start
 
 
-ansible -i "127.0.0.1," -c local -v -m postgresql_user -U postgres -a "name=awx-dev password=AWXsome1 login_user=postgres login_host=postgres" all
-ansible -i "127.0.0.1," -c local -v -m postgresql_db -U postgres -a "name=awx-dev owner=awx-dev login_user=postgres login_host=postgres" all
+ansible -i "127.0.0.1," -c local -v -m postgresql_user --become-user postgres -a "name=awx-dev password=AWXsome1 login_user=postgres login_host=postgres" all
```

Comment:
@ryanpetrello - In the course of testing this PR against ansible's devel branch, found that the `-U` flag was [deprecated](https://gist.github.com/jladdjr/fbf24d56c6f63071bb177e896998c490#file-gistfile1-txt-L41-L44). Can move this to a separate PR instead of pork-barreling if you'd like.

Reply:
It's fine to have this here in my opinion.

Reply:
Agreed, keep it here.